packages/plugins/plugin-table/src/components/ObjectTable/ObjectTable.tsx

```diff
@@ -29,9 +29,15 @@ export type ObjectTableProps = Pick<TableProps<any>, 'stickyHeader' | 'role'> &
 export const ObjectTable = ({ table, role, stickyHeader }: ObjectTableProps) => {
   const space = getSpace(table);
   const [showSettings, setShowSettings] = useState(false);
-
   useEffect(() => setShowSettings(!table.schema), [table.schema]);
 
+  const [schemas, setSchemas] = useState<DynamicSchema[]>([]);
+  useEffect(() => {
+    if (space) {
+      void space.db.schema.list().then(setSchemas).catch();
```

Comment:
todo don't don't void promise in useEffect

Reply:
why?